cli/src/modules/pskb.rs

```diff
@@ -0,0 +1,259 @@
+                        );
+
+                        // Sweep UTXO set.
+                        match unlock_utxos_as_pskb(references, &receive_address, script_sig, priority_fee_sompi as u64) {
+                            Ok(pskb) => {
+                                let pskb_hex = pskb.serialize()?;
+                                tprintln!(ctx, "{pskb_hex}");
+                            }
+                            Err(e) => tprintln!(ctx, "Error generating unlock PSKB: {}", e.to_string()),
+                        }
+                    }
+                    "sign" => {
+                        let pskb = Self::parse_input_pskb(argv.first().unwrap().as_str())?;
+
+                        // Sign PSKB using the account's receiver address.
+                        match account.pskb_sign(&pskb, wallet_secret.clone(), payment_secret.clone(), Some(&receive_address)).await {
+                            Ok(signed_pskb) => {
+                                let pskb_pack = String::try_from(signed_pskb)?;
+                                tprintln!(ctx, "{pskb_pack}");
+                            }
+                            Err(e) => terrorln!(ctx, "{}", e.to_string()),
+                        }
+                    }
+                    "address" => {
+                        tprintln!(ctx, "\r\nP2SH address: {}", script_p2sh);
+                    }
+                    v => {
+                        terrorln!(ctx, "unknown command: '{v}'\r\n");
+                        return self.display_help(ctx, argv).await;
+                    }
+                }
+            }
+            "sign" => {
+                if argv.len() != 1 {
+                    return self.display_help(ctx, argv).await;
+                } else {
+                    let (wallet_secret, payment_secret) = ctx.ask_wallet_secret(None).await?;
+                    let pskb = Self::parse_input_pskb(argv.first().unwrap().as_str())?;
+
+                    match account.pskb_sign(&pskb, wallet_secret.clone(), payment_secret.clone(), None).await {
+                        Ok(signed_pskb) => {
+                            let pskb_pack = String::try_from(signed_pskb)?;
+                            tprintln!(ctx, "{pskb_pack}");
+                        }
+                        Err(e) => terrorln!(ctx, "{}", e.to_string()),
+                    }
+                }
+            }
+            "send" => {
+                if argv.len() != 1 {
+                    return self.display_help(ctx, argv).await;
+                } else {
+                    let pskb = Self::parse_input_pskb(argv.first().unwrap().as_str())?;
+
+                    match account.pskb_broadcast(&pskb).await {
+                        Ok(sent) => tprintln!(ctx, "Sent transactions {:?}", sent),
+                        Err(e) => terrorln!(ctx, "Send error {:?}", e),
+                    }
+                }
+            }
+            "debug" => {
```

Comment:
Rather than `debug` this probably should be `parse`. My thinking is that parsing a PSKB is a normal thing that someone would do to check out it's contents before trying to sign it or do anything else with it (if they receive it from someone else). So in that sense, they're not really trying to debug anything - they're just doing due diligence. `debug` implies you're trying to investigate something.

Reply:
@1bananagirl ^ WDYT?  

I think if `parse` is introduced, it should present a formatted view (nice layout) while debug can remain as a debug view.

Reply:
Good idea. Does the parse view also have all fields displayed, or a limited range of ?

Reply:
Added in PR  https://github.com/aspectron/rusty-kaspa/pull/92 @coderofstuff @aspect

Reply:
This has been merged ^.